notebooks/ts_generic_example.ipynb

```diff
@@ -246,9 +238,94 @@
     "print(run.data)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b0aa46b5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Show the results\n",
+    "\n",
+    "result = run.poll()\n",
+    "print(result.data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "36284c02",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Import Packages for inference\n",
+    "\n",
+    "from tq42.algorithm import (\n",
+    "    GENERIC_ML_INFER,\n",
+    "    TSEvalInputsProto,\n",
+    "    ModelStorageInfoProto,\n",
+    "    GenericMLInferMetadataProto,\n",
+    "    GenericMLInferParametersProto,\n",
+    "    InferDataProcessingParametersProto) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "677e558f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set the model for running inference\n",
+    "\n",
+    "classical_eval_message=MessageToDict(GenericMLInferMetadataProto(\n",
+    "    parameters=GenericMLInferParametersProto(\n",
+    "        data_processing_parameters=InferDataProcessingParametersProto(\n",
+    "            input_columns=[0, 1, 2, 3], timestamp_columns=[]\n",
+    "        )\n",
+    "    ),\n",
+    "    inputs=TSEvalInputsProto(\n",
+    "        model=ModelStorageInfoProto(storage_id=\"ENTER_MODEL_STORAGE_ID_HERE\"),\n",
```

Comment:
why does the user need to input the model id? Couldn't you just derive it from the training experiment run??

Reply:
used `run.data.id`